### PR TITLE
Fix: Convert the default instruction text styling from italics to bold

### PR DESCRIPTION
**Pull request by StuartNicholls**
Update instruction styling to 'normal' and weight to 'bold'. Update variables in menu & page header

Fixes: #395

### CODE REVIEW

**Review by swashbuck — APPROVED** [review]
👍

**Review by kirsty-hames — APPROVED** [review]

File: less/_defaults/_font-config.less [hunk 1]
```diff
@@ -121,7 +121,7 @@
 
 @page-instruction-size: @page-body-size;
 @page-instruction-family: @body-family;
```
Comment (guywillis) [inline less/_defaults/_font-config.less:123]: Can you update to `@page-instruction-family: @page-body-family;` to keep inline with `@menu-instruction-family` please

**Review by guywillis — APPROVED** [review]
Thanks!

File: less/_defaults/_font-config.less (hunk 1 above)
Comment (guywillis) [reply less/_defaults/_font-config.less]: Good spot

**Review by kirsty-hames — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**